types/rxjs-exec/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for rxjs-exec 0.4
+// Project: https://github.com/kssfilo/rxjs-exec
+// Definitions by: Meno Abels <https://github.com/mabels>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as Rx from 'rxjs';
+
+export class Options {
+    binary?: boolean;
+    input?: string|Buffer;
+    stdin?: NodeJS.WritableStream;
```

Comment:
stdin passed needs to be Boolean ? https://github.com/kssfilo/rxjs-exec#connecting-processstdin